src/screens/hardware-wallets/constants.tsx

```diff
@@ -0,0 +1,3 @@
+import * as i18n from '@/languages';
+
+export const TRANSLATIONS = i18n.l.hardware_wallets;
```

Comment:
I like the thinking here but also I'm kinda with Kuba that the dumber option of just letting IDEs complete the full path everywhere is almost easier. Don't think we need to change this though, thanks for thinking deeper about things like this!

Reply:
ya thats fair, tbh i just did this because for me this was easier than repeatedly typing `i18n.l.hardware_wallets`